src/src/PluginStructs/P121_data_struct.cpp

```diff
@@ -0,0 +1,37 @@
+#include "../PluginStructs/P121_data_struct.h"
+
+#ifdef USES_P121
+
+// Needed also here for PlatformIO's library finder as the .h file
+// is in a directory which is excluded in the src_filter
+#include <Adafruit_Sensor.h>
+#include <Adafruit_HMC5883_U.h>
+
+bool P121_data_struct::begin(bool initSettings)
```

Comment:
`initSettings` is not used. Is it still needed for something you're perhaps working on?